velox/vector/BuilderTypeUtils.h

```diff
@@ -53,8 +54,9 @@ constexpr bool isIntegral() {
  */
 template <typename T>
 constexpr bool admitsDictionary() {
-  return std::is_same_v<T, int64_t> || std::is_same_v<T, double> ||
-      std::is_same_v<T, StringView>;
+  return std::is_same_v<T, int128_t> || std::is_same_v<T, int64_t> ||
+      std::is_same_v<T, double> || std::is_same_v<T, StringView> ||
+      std::is_same_v<T, velox::Timestamp>;
```

Comment:
Shall we have a test for this if we enable dictionary encoding on timestamp type in velox/vector/tests/VectorTest.cpp? thanks!

Reply:
This code is currently unused in velox and is not used to determine permissible encodings. It may be better to remove this from velox to avoid such confusion.